### PR TITLE
Another try to revive Infection gamemode

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

Actually yes. This gamemode is very cool, basicaly it is a blob with more structures, more interesting servants, interesting legendary weapon and boss system, and more fun things. This gamemode was originaly on tg, and was abadoned. But actually it doesnt deserve to be forgoten, and is fucking godlike.

# Notes

Yes, I am trying with minimal ss13 coding experience to revive so complex and ambitious gamemode, that was abadoned by its original creator, by another guy who alredy tried to revived it, and doesnt exist on ANY other server. But for me YogStation is a blessed place, where forgotten and underrated, but still very good gamemodes(clockwork cult, shadowlings, gangsters) get a second chance to live. So I hope that the infection gamemode would be also revived here, since it has very interesting mechanics  and a good idea. And i am not very sure that i can maintain it only by myself, so if you have some ideas, tips or some code suggestions, you are always welcome. 

# Wiki Documentation

Create a wiki page about this antag.

# Changelog

:cl:  
rscadd: Tries to revive Infection gamemode
soundadd: Adds sounds for it
imageadd: Adds sprites for it 
/:cl:

### CODE REVIEW

**Comment by Simplehorror** [comment]
hand of god next

**Comment by ghost** [comment]
Are you joking, fucking merge conflicts in a 86 files changed PR...

**Comment by ghost** [comment]
> hand of god next

Yes.

**Comment by Simplehorror** [comment]
> Are you joking, fucking merge conflicts in a 86 files changed PR...

You touched every map

**Comment by ghost** [comment]
> > Are you joking, fucking merge conflicts in a 86 files changed PR...
> 
> You touched every map

Yes, since this gamemode needs beacons to be placed, or it wouldn't work.

**Comment by ghost** [comment]
> Are you joking, fucking merge conflicts in a 86 files changed PR...

Actually i didnt think that this PR would be doomed so fast.

**Comment by ghost** [comment]
I fucking hate merge conflicts i dont want to do all the porting work again but with an updated files

**Comment by ghost** [comment]
Dead for now

File: html/changelog.html [hunk 1]
```diff
@@ -1515,6 +1515,18 @@ <h3 class="author">adamsong updated:</h3>
...
+			<h3 class="author">ToasterBiome updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="rscadd">add an extra air pump to toxins</li>
+				<li class="tweak">moves paper bin one tile to the right</li>
+				<li class="tweak">moved sci wardrobe vending machine to experimentor room</li>
+				<li class="rscadd">add camera to incinerator and incinerator access</li>
+				<li class="bugfix">fix rogue disposal pipe in theatre</li>
+				<li class="bugfix">fix rogue power cables on the floor of botany backroom (power comes from the other way)</li>
+				<li class="tweak">extend botany backroom so you can access the wardrobe without opening the locker</li>
+			</ul>
```
Comment (adamsong) [inline html/changelog.html:1529]: Revert this

File: html/changelogs/.all_changelog.yml [hunk 2]
```diff
@@ -30821,14 +30821,3 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
...
-  Altoids:
-  - bugfix: People who have died from traumatic brain injury no longer appear happy
-      and healthy on crew monitoring consoles.
-  - bugfix: It is now possible for the pool in dorms to be closed due to singularity.
-  - tweak: Mobs which would be electrocuted were they to enter a pool now receive
-      a small warning about that when they examine the pool turfs.
-  - spellcheck: Having your mood changed by going into a pool no longer reduces your
-      spelling abilities.
-  ToasterBiome:
-  - rscadd: Nmajask is in the game now
```
Comment (adamsong) [inline html/changelogs/.all_changelog.yml:30834]: Revert this

**Review by adamsong — CHANGES_REQUESTED** [review]

**Comment by bruhlookatthisdood** [comment]
afton-ending-afton

**Comment by Bibby0110** [comment]
Reopen when not dead :)

**Comment by ghost** [comment]
E

**Pull request closed without merging**